cloudformation/membership-attribute-service.json

```diff
@@ -165,6 +166,42 @@
               },
               {
                 "Effect": "Allow",
+                "Action": "dynamodb:PutItem",
+                "Resource": {
+                  "Fn::FindInMap": ["StageVariables", {
+                    "Ref": "Stage"
+                  }, "DynamoDBTableTestUsers"]
+                }
+              },
+              {
+                "Effect": "Allow",
+                "Action": "dynamodb:GetItem",
```

Comment:
There seems to be a fair bit of repetition for the Actions - can this be more concisely expressed? I think you should be able to group the Actions like this:

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "dynamodb:PutItem",
                "dynamodb:UpdateItem",
                "dynamodb:DeleteItem"
            ],
            "Resource": "arn:aws:dynamodb:us-west-2:123456789012:table/Books"
        }
    ]
}
```

http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ExamplePolicies_DDB.html#d0e44734